src/Clide.Interfaces/AwaitableValue.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace Clide
+{
+    /// <summary>
+    /// Provides a factory for <see cref="AwaitableValue{T}"/>.
+    /// </summary>
+    public static class AwaitableValue
```

Comment:
What about calling this factory (just the factory) as simply “Awaitable”?
So, the API usage ends like this:

var awaitableValue = Awaitable.Create(...);

I thinks this is shorter and more idiomatic.

Reply:
I like it! Fixing it.

Reply:
Should the value itself also be just `Awaitable<T>` then?